.env.dist

```diff
@@ -4,3 +4,4 @@ APP_SECRET=550847c0e8dde7b2029df32625881bb1
 #TRUSTED_PROXIES=127.0.0.1,127.0.0.2
 #TRUSTED_HOSTS=localhost,example.com
 ###< symfony/framework-bundle ###
+COMPOSE_PROJECT_NAME=dummyapi
```

Comment:
would rather not upgrade `docker-compose` on every single build

Reply:
(to allow `dummy-api`)

Reply:
What's the versioning problem?

Reply:
Older versions of `docker-compose` strip `-` from container names, while newer versions allow for `dummy-api_fpm_1` container names. Travis CI has one of the older ones.

Reply:
Maybe could be in `.travis.yml` instead?

Reply:
Yes, it should be equivalent. In general would default to the folder name without it.